miscelanea/002-init-container/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM docker.io/freeipa/freeipa-server:centos-8
+
+# Workaround for running systemd mode from podman
+RUN mkdir /usr/sbin/original \
+    && ln -svf /usr/lib/systemd/systemd /usr/sbin/original/init \
+    && rm /usr/sbin/init
+COPY init-wrapper.sh /usr/sbin/init
+COPY init-data /usr/loca/sbin/init
+
+# Workaround to don't loose the machine-id between different executions
```

Comment:
This is not needed. False positive when I got the report from dive tool. I will remove this block.